docs/REST API/rest-api.md

```diff
@@ -4721,6 +4623,27 @@ To wipe a macOS, iOS, iPadOS, or Windows host, the host must have MDM turned on.
 }
 ```
 
+### Cancel host's upcoming activity
+
+`DELETE /api/v1/fleet/hosts/:id/activities/upcoming/:upcoming_activity_id`
+
+#### Parameters
+
+| Name | Type    | In   | Description                  |
+| ---- | ------- | ---- | ---------------------------- |
+| id   | integer | path | **Required**. The host's ID. |
+| upcoming_activity_id   | string | path | **Required**. The ID of host's upcoming activity. |
```

Comment:
I'd consider shortening this to just `activity_id` since we already have `/upcoming/` in the URL. Thoughts?
```suggestion
| activity_id   | string | path | **Required**. The ID of host's upcoming activity. |
```